src/components/PokeCard.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from "react";
+import { getPokemonData } from "../services";
+import { Title } from "./Title";
+import { Image } from "./Image";
+import { Link } from "./Link";
+
+export const PokerCard = () => {
+  const [pokeName, setPokeName] = useState("");
+  const [spriteLinks, setSpriteLinks] = useState([]);
+  const [pokeVideo, setPokeVideo] = useState("");
+
+  useEffect(() => {
+    const getPokemonHandler = async () => {
+      const {
+        name,
+        sprites: { front_default, front_shiny },
```

Comment:
keep variables on camelCase instead, i.e:
```suggestion
        sprites: { front_default: frontDefault, front_shiny: frontShiny },
```

Reply:
@wdonet you are very right here, I always try to follow the same convention in my code I miss this case. thanks